hbase-server/src/main/jamon/org/apache/hadoop/hbase/tmpl/master/RegionVisualizerTmpl.jamon

```diff
@@ -33,7 +33,7 @@ limitations under the License.
             format: { type: 'json', property: 'data' }
           },
           transform: [
-            { calculate: "split(datum.server_name.servername, ',', 1)[0]", as: 'server_name' },
+            { calculate: "split(datum.server_name.server_name, ',', 1)[0]", as: 'server_name' },
```

Comment:
I'm very surprised to see this change. Was there an unrelated change to the JSON serialization or the underlying datatype that caused this divergence?

Reply:
@ndimiduk sir.
I am sorry, I looked up the commit history, but I didn't find the commit about the change of this problem.